frontend/src/app/modules/availability/new-slot/questions/questions.component.sass

```diff
@@ -35,15 +55,18 @@ $question-input-width: 337px
         padding: $container-padding-top-bottom 14px
 
 .list-item-wrapper
-    padding-bottom: 20px
+    padding-top: 5px
+    padding-bottom: 5px
 
 .question-index
     margin-right: 17px
+    font-weight: 700
```

Comment:
```suggestion
```

Repeating css properties in multiple classes should be spit into modular pieces, if possible. If you don't like using .bold class explicitly in html elements, please, factor out font-weight:700 into separate place and comma-enumerate all classes/elements, which need it.